src/popup.ts

```diff
@@ -66,32 +47,20 @@ const chkbox = [
 async function saveData(thisConfig: ConfigInterface) {
     try {
         await browser.storage.sync.set(thisConfig as any);
-        document.getElementById('status').innerHTML = "저장완료. " + escapeHtml(JSON.stringify(config));
-        setTimeout(
-            () => {
-                document.getElementById('status').innerHTML = "";
-            },
-            750
-        )
+        bgconsole.log(`저장완료. ${JSON.stringify(config)}`);
     } catch (e) {
-        document.getElementById('status').innerHTML = "저장실패. " + escapeHtml(JSON.stringify(config));
-        setTimeout(
-            () => {
-                document.getElementById('status').innerHTML = "";
-            },
-            1500
-        )
+        bgconsole.error(`저장 실패. ${JSON.stringify(config)}`);
     }
 }
 
 function setHook() {
     for (const chk of chkbox) {
-        console.log(chk);
         const datasetVal = chk.dataset.val as keyof ConfigInterface;
         if (chk.type === "checkbox") {
             chk.checked = config[datasetVal] as boolean;
         } else if (chk.type === "text" || chk.type === "url") {
             chk.value = config[datasetVal] as string;
+            chk.value = chk.value === 'undefined' ? '' : chk.value;
```

Comment:
Isn't this should be `typeof chk.value === 'undefined'` ?

Reply:
and could you open and close Parentheses `()` for better readability?

Reply:
How About
```TS
chk.value = typeof(config[datasetVal]) !== 'undefined' ? config[datasetVal] as string : '';
```?

Reply:
Good